docs/i2c.rst

```diff
@@ -8,6 +8,11 @@ using the I²C bus protocol. There can be multiple slave devices connected at
 the same time, and each one has its own unique address, that is either fixed
 for the device or configured on it. Your board acts as the I²C master.
 
+We use 7-bit addressing for devices because of the reasons stated
+`here <http://www.totalphase.com/support/articles/200349176-7-bit-8-bit-and-10-bit-I2C-Slave-Addressing>`_.
+
```

Comment:
Yes. I think it important to accentuate that there's a possibly unexpected difference. The alternative was to emphasise it but it looked a bit naff IMHO.